authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -103,10 +103,7 @@ protected RangerAuthorizationPlugin(String metalake, Map<String, String> config)
     rangerServiceName = config.get(RangerAuthorizationProperties.RANGER_SERVICE_NAME);
     rangerClient = new RangerClientExtension(rangerUrl, authType, rangerAdminName, password);
 
-    if (Boolean.parseBoolean(
-        config.get(RangerAuthorizationProperties.RANGER_SERVICE_CREATE_IF_ABSENT))) {
-      createRangerServiceIfNecessary(config, rangerServiceName);
-    }
+    createRangerServiceIfNecessary(config, rangerServiceName);
```

Comment:
Why do we need delete `if` sentence?

Reply:
I put the `if` sentence into the createRangerServiceIfNecessary method and judge the config option and service existence.

Reply:
OK